odm2api/ODM2/services/readService.py

```diff
@@ -875,7 +887,7 @@ def getDataSetsValues(self, ids=None, codes=None, uuids=None, dstype=None):
         return None
 
 
-    def getSamplingFeatureDatasets(self, ids=None, codes=None, uuids=None, dstype=None):
+    def getSamplingFeatureDatasets(self, ids=None, codes=None, uuids=None, dstype=None, type=None):
```

Comment:
`type` is a reserved python word. Please use `sftype` to be clear, and avoid using reserved word.

Reply:
Also, the doc string doesn't seem to reflect this additional argument. Please add.